### PR TITLE
Update subfields.yaml

### PR DESCRIPTION
**Pull request by kwanliang556677**
Add new subfield for digital building name points

### CODE REVIEW

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 1]
```diff
@@ -41,7 +43,14 @@ component:
...
   wheel: "Component used for transfer of heat from incoming to outgoing air streams. "
-descriptor:
+descriptor: 
+  emergency : "a situation that poses an immediate risk to health, life property,  or environment."
+  scene : "a combination of objects or events in view or happening of an event that attracts attention."
+  leak : "Media leaving the system abnormally."
+  detection : "Process of identifying conditions."
+  fire : "a potential to cause physical damage through burning"
+  suppression : "a range of firefighting tactics used to suppress fire"
+  Photocatalyst : "a material that uses absorbed light energy (usually UV) to drive desired reactions that would not otherwise proceed under ambient conditions"
```
Comment (charbull) [inline ontology/yaml/resources/subfields/subfields.yaml:53]: can you please make it low capital letter: Photocatalyst ==> photocatalyst.

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 2]
```diff
@@ -152,6 +161,7 @@ descriptor:
   winter: "Method or process used during colder weather (i.e. winter season)."
   zone: "Region of building which is conditioned."
 measurement:
+  fault : "Signal that a fault (low/medium) priority warning is present; can compromise system operation, but not necessarily imminently."
```
Comment (charbull) [inline ontology/yaml/resources/subfields/subfields.yaml:164]: fault does not belong here, it is not a `measurement`.
can you please move it under `descriptor`?

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 3]
```diff
@@ -18,6 +18,8 @@ aggregation:
   min: "Minimum value (e.g. Min_Ventilation_Air_Flow_Setpoint)"
   total: "Sum total of some set of values (e.g. total_request_heating_count)"
 component:
+  tank : "Storage device for media."
+  lighting : "artificial light source devices or natural illumination by capturing daylight."
```
Comment (charbull) [inline ontology/yaml/resources/subfields/subfields.yaml:22]: lighting does not belong here. There will be work on lighting in another PR.

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 4]
```diff
@@ -41,7 +43,14 @@ component:
   valve: "Component which meters the flow of water within a system or device."
   vane: "Component for guiding media flow."
   wheel: "Component used for transfer of heat from incoming to outgoing air streams. "
-descriptor:
+descriptor:
```
Comment (tasodorff) [inline ontology/yaml/resources/subfields/subfields.yaml:46]: can the subfields defined below please be added in alphabetical order to allow for readability upon commit?

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 2 above)
Comment (tasodorff) [reply ontology/yaml/resources/subfields/subfields.yaml:164]: in general, nothing should go under measurements without some type of units asociated with it (e.g. `temperature` --> K, F, etc.).

Additionally: the description for `fault` is similar to what a definition would be for `alarm` (which is currently absent from the ontology). Would there be a distinction between these values in practice, or are all `faults` just `alarms`?

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 1 above)
Comment (kwanliang556677) [reply ontology/yaml/resources/subfields/subfields.yaml]: Ok, changes commited.

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 2 above)
Comment (kwanliang556677) [reply ontology/yaml/resources/subfields/subfields.yaml]: fault has been moved to measurement_descriptor.

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 3 above)
Comment (kwanliang556677) [reply ontology/yaml/resources/subfields/subfields.yaml]: Lightning has been removed.

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 4 above)
Comment (kwanliang556677) [reply ontology/yaml/resources/subfields/subfields.yaml:45]: ok, I have rearrange the order.

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 3 above)
Comment (kwanliang556677) [reply ontology/yaml/resources/subfields/subfields.yaml]: Can we know when the lightning PR be merged into master branch? Or should we add it in our branch?

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 5]
```diff
@@ -41,7 +43,14 @@ component:
   valve: "Component which meters the flow of water within a system or device."
   vane: "Component for guiding media flow."
   wheel: "Component used for transfer of heat from incoming to outgoing air streams. "
-descriptor:
+descriptor: 
+  emergency : "a situation that poses an immediate risk to health, life property,  or environment."
+  scene : "a combination of objects or events in view or happening of an event that attracts attention."
```
[the remark this answers is not in the document]
Comment (kwanliang556677) [reply ontology/yaml/resources/subfields/subfields.yaml]: This subfield is proposed specially for lighting system. Lighting scene setting is used to dimming up/down individual channels, to create a tailor made lighting scene

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 3 above)
Comment (charbull) [reply ontology/yaml/resources/subfields/subfields.yaml]: adding @pisuke who is working on this part.

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 5 above)
[the remark this answers is not in the document]
Comment (tasodorff) [reply ontology/yaml/resources/subfields/subfields.yaml]: Still not sure I understand the point of this. Is a scene (in this context) the area served by multiple coordinated lighting systems, where illumination can be coordinated to focus attention on certain features (e.g. a podium or stage) in the area?

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 3 above)
Comment (tasodorff) [reply ontology/yaml/resources/subfields/subfields.yaml]: is this still ongoing?

**Comment by charbull** [comment]
@tasodorff did you run the validator? just to make sure before I merge.

I need to set up the validator in github actions. (pending)

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 3 above)
Comment (JennyChan-ptpl) [reply ontology/yaml/resources/subfields/subfields.yaml]: @pisuke are you going to add subfield of "Lighting"? we need this to generate lighting point naming.
